web/src/components/tasks/xray-registration/XrayRegistrationSummary.test.tsx

```diff
@@ -0,0 +1,100 @@
+import { XrayRegistrationSummary } from "@/components/tasks/xray-registration/XrayRegistrationSummary";
+import { getMergedConfig } from "@/contexts/configContext";
+import { getCurrentDate } from "@businessnjgovnavigator/shared/dateHelpers";
+import { XrayRegistrationStatus } from "@businessnjgovnavigator/shared/xray";
+import { fireEvent, render, screen } from "@testing-library/react";
+import dayjs from "dayjs";
+import localizedFormat from "dayjs/plugin/localizedFormat";
+
+const Config = getMergedConfig();
+
+describe("<XrayRegistrationSummary />", () => {
+  dayjs.extend(localizedFormat);
+  const futureDate = getCurrentDate().add(2, "month").format("L");
+  const fifteenDaysInTheFutureDate = getCurrentDate().add(15, "days").format("L");
+  const pastDate = getCurrentDate().subtract(2, "month").format("L");
```

Comment:
[dust] Are we covering for situations of running this test toward the end of the month / toward the end of the year so we don't introduce date-based flakiness with tests? I know this has been an issue in the past in our codebase.

Reply:
I checked end of month and end of the year by changing the value of current date in both the code and tests manually to ensure it returned the right results. I don't think these tests will flake. I can also add cases for where the currentDate has different scenarios such as [end of month, end of year, current] by overwriting what the currentDate function returns? I would however want to do that as follow up to push this work forward. Thoughts?